ConsoleApp/Library/Customer.cs

```diff
@@ -0,0 +1,12 @@
+namespace ConsoleApp.Library;
+
+class Customer
+{
+    public string Name {get; init;}
+    public enum _sex
```

Comment:
code convention for enum - always pascal case
```suggestion
    public enum Sex
```